com.salesforce.ide.ui.editors/src/com/salesforce/ide/ui/editors/apex/assistance/ApexCompletionProcessor.java

```diff
@@ -77,15 +75,6 @@ public boolean updatePresentation(int documentPosition, TextPresentation present
     //  C O N S T R U C T O R S
     public ApexCompletionProcessor() {}
 
-    //   M E T H O D S
```

Comment:
I needed to reuse the editor configuration to provide syntax highlighting in the template preference page and template editor. The editor configuration uses the completion processor (which does not work at this time). The completion processor had an unused reference to a project but the project is not available during template management. Since the project is unused, I removed it.